atd-etl/cris_import/cris_import.py

```diff
@@ -190,12 +204,20 @@ def get_secrets():
     return onepasswordconnectsdk.load_dict(client, REQUIRED_SECRETS)
 
 
-def specify_extract_location(file):
+def specify_extract_location():
+    zip_files = glob.glob('/app/development_extracts/*.zip')
+    if not zip_files:
+        return False
+
     zip_tmpdir = tempfile.mkdtemp()
-    shutil.copy(file, zip_tmpdir)
+
+    for file_to_copy in zip_files:
+        shutil.copy(file_to_copy, zip_tmpdir)
+
     return zip_tmpdir
```

Comment:
I actually have used this technique in the past. This spiffs up an existing `specify_extract_location()` function.